licenses/models.py

```diff
@@ -344,3 +416,58 @@ def sampling_plus(self):
     @property
     def include_share_adapted_material_clause(self):
         return self.license_code in ["by", "by-nc"]
+
+    def tx_upload_messages(self):
+        """
+        Upload the messages to Transifex,
+        creating the resource if it doesn't already exist.
+        """
+        # Have to do English first, they get uploaded differently as the "source" messages
+        # and are required if we need to first create the resource in Transifex.
+        en_legalcode = self.get_legalcode_for_language_code(DEFAULT_LANGUAGE_CODE)
+        helper = TransifexHelper()
+        helper.upload_messages_to_transifex(legalcode=en_legalcode)
+        for legalcode in self.legal_codes.exclude(language_code=DEFAULT_LANGUAGE_CODE):
+            helper.upload_messages_to_transifex(legalcode=legalcode)
+
+
+class TranslatedLicenseName(models.Model):
+    license = models.ForeignKey(License, related_name="names", on_delete=models.CASCADE)
+    language_code = models.CharField(
+        max_length=MAX_LANGUAGE_CODE_LENGTH,
+        help_text="E.g. 'en', 'en-ca', 'sr-Latn', or 'x-i18n'. Case-sensitive?",
+    )
+    name = models.CharField(max_length=250, help_text="Translated name of license")
+
+    class Meta:
+        unique_together = [
+            ("license", "language_code"),
+        ]
+
+    def __str__(self):
+        return f"TranslatedLicenseName<{self.language_code}, {self.license}>"
+
+
+class TranslationBranch(models.Model):
+    branch_name = models.CharField(max_length=40)
+    legalcodes = models.ManyToManyField("LegalCode")
+    version = models.CharField(
+        max_length=3, help_text="E.g. '4.0'. Not required.", blank=True, default=""
+    )
+    language_code = models.CharField(
+        max_length=MAX_LANGUAGE_CODE_LENGTH,
+        help_text="E.g. 'en', 'en-ca', 'sr-Latn', or 'x-i18n'. Case-sensitive?",
+    )
+    last_transifex_update = models.DateTimeField(
```

Comment:
does this field need to be tz aware? Or will the transifex api return utc time?

Reply:
I'm just wondering in general what `time` we need to be tracking in the project and then displaying as a part of the translation status page. 

More of a question than a fix I'm requesting.